### DISCUSSION TITLE
MAUI is not getting the funding it deserves

### DISCUSSION THREAD

**Discussion by akhanalcs** [dotnet/maui#10039] [General]
MAUI has enormous potential. Imagine Microsoft attracting a sizable number of new developers into .NET and MAUI ecosystem using MAUI. 
With that strength, Microsoft can make all kinds of strategic moves; they can even launch their own phone OS someday and there would be more quality apps in the windows store and so on.
That way Microsoft can get new consumers into their other products and services like Surface, Azure, Azure DevOps, Teams and so on.

It is beyond me as to why leadership at Microsoft is failing to see this potential in MAUI and not funding it properly. 
It looks like there're only 5 developers at Microsoft who work on MAUI. 
The velocity at which MAUI bugs are being fixed is not sustainable for a successful product. 

MAUI needs more developers. 
If Microsoft has money to spend billions of dollars on buying game studios, they sure can afford to hire more developers!

**Comment by Majestic-Mustang** [comment]
You're right. Satya Nadella is a follower, not a leader and he's doing nothing but stifling innovation at Microsoft.
He's failing to see the potential in products like MAUI, Windows on ARM etc.

Satya needs to go!

**Comment by rlalance** [comment]
100%, this could be a complete game changer!
